packages/manager/modules/pci/src/projects/project/creating/creating.controller.js

```diff
@@ -0,0 +1,30 @@
+export default class ProjectCreatingCtrl {
+  /* @ngInject */
+  constructor(Poller) {
+    this.Poller = Poller;
+  }
+
+  startCreationPolling() {
+    return this.Poller.poll(`/cloud/project/${this.projectId}`, null, {
+      namespace: 'pci.projects.creating',
+      successRule(details) {
+        return details.status === 'ok';
+      },
+    })
+      .then(() => this.onProjectCreated());
+  }
+
+  /* =====================================
+  =            Initialization            =
+  ====================================== */
+
+  $onInit() {
+    this.Poller.kill({
```

Comment:
We used to kill poller even on `$onDestroy` hook.

Is that make sense here too?

Reply:
Isn't that a `Promise` ?